compiler/rustc_hir_analysis/src/lib.rs

```diff
@@ -233,6 +222,17 @@ pub fn check_crate(tcx: TyCtxt<'_>) {
         }
     });
 
+    if tcx.features().rustc_attrs() {
```

Comment:
Why dump attrs stuff later? I'm not opposed, just there's no motivation given.

Reply:
I wanted to make space there to merge the parallel blocks before and after